domains/mail.is-a-backend.dev.json

```
@@ -0,0 +1,14 @@
+{
+    "domain": "is-a-backend.dev",
+    "subdomain": "mail",
+
+    "owner": {
+        "email": "sinzie@pm.me"
+    },
+
+    "records": {
+        "A": ["38.87"]
```

Please use a valid record here.

Are you wanting to use the same record as most of the other domains?
```suggestion
        "A": ["38.87.162.97"]
```